### ISSUE TITLE
Branch coverage

### ISSUE THREAD

**Issue by PoOwAa** [zgosalvez/github-actions-report-lcov#11]
Currently I can't generate branch coverage. There are some scenario, when coverage is 100%, but branch coverage is lower.

From lcov 1.10 this feature needs a new flag: `--rc lcov_branch_coverage=1`

Maybe we should extends the functionality of the action with this option.

**Comment by zgosalvez** [comment]
This action generates a summary as a comment. It includes branches, as you can see in the readme. Are you referring to the HTML report generated? I want to make sure since that's the part of the code that you PR-ed.

**Comment by PoOwAa** [comment]
@zgosalvez that's right because the comment is generated from the HTML report. As I mentioned above, the generated HTML does not contain the branch coverage from lcov 1.10, it should be enabled via the flag.

**Comment by zgosalvez** [comment]
I've merged your PR. Thanks! LMK if there's an issue with the latest release. Otherwise, feel free to close this issue :)
